### DISCUSSION TITLE
支持各种服务端下载，添加开服功能

### DISCUSSION THREAD

**Discussion by QinQiuFox** [Meloong-Git/PCL#1704] [提问] [answered]
建议加入内置下载原版服、插件服、模组服，并且加入开服功能

（觉得没必要的不要评论喷人！！！！！，我知道有各种镜像站，但不觉得这样更方便吗？）

**Comment by MoYuan-CN** [comment, marked as answer]
首先我要提出的是 PCL2 是 Minecraft **启动器** 而不是 开服器
原版服务端的下载加入已在 #112 中提出，你可以到 #113 为其投票！
其他的功能（内置下载插件服、模组服，并且加入开服功能）则不会制作，这太超出 PCL2 的范围了

**Comment by Super-XiaoNian** [comment]
没必要啦，龙猫还是个大学生，眼前有n倍的issues+投票功能要去做，也有学业的事情，没太多精力去搞这些，

**Reply by Chiloven945** [reply]
龙猫已经毕业了捏（

**Reply by iPodtouch6557** [reply]
其实有一点issue压箱底会显得这玩意超级火……
只有那几个issue会显得很冷清

**Reply by Super-XiaoNian** [reply]
我很期待Log日志那个）
龙猫咋时候把游戏日志实际窗口给搞好

**Comment by huhujieshuo** [comment]
推荐开服器：MSL
